tests/test_flows/test_flow.py

```diff
@@ -67,6 +67,19 @@ def test_get_flow(self):
         self.assertEqual(subflow_3.parameters['L'], '-1')
         self.assertEqual(len(subflow_3.components), 0)
 
+    def test_tagging(self):
+        flow = openml.flows.get_flow(4024)
```

Comment:
That's dangerous. If the server is purged, this flow will no longer exist. Maybe you should just query the server for a flow and take the first.

Reply:
This flow is used in another test, which is why I selected it ;) We can have the setup query for a flow and then use that everywhere, though.

Reply:
Though the server was purged and it didn't fail, right?

Reply:
But that flow is loaded from the production server.

Reply:
Querying for a flow would be better.

Reply:
lol it's actually using the production server in the other test...